apps/ejabberd/src/mod_roster_mnesia.erl

```diff
@@ -56,14 +56,10 @@ read_roster_version(LUser, LServer) ->
         [] -> error
     end.
 
-write_roster_version(LUser, LServer, InTransaction, Ver) ->
-    US = {LUser, LServer},
-    if InTransaction ->
-           mnesia:write(#roster_version{us = US, version = Ver});
-       true ->
-           mnesia:dirty_write(#roster_version{us = US,
-                                              version = Ver})
-    end.
+write_roster_version(LUser, LServer, true, Ver) ->
```

Comment:
All right, spec will help here. Maybe it's not worth changing the boolean to a meaningful atom for this code.

In general I think in many places where we need a boolean we can have a meaningful atom. Just an idea to think about :)